plugin/telegram/chat.go

```diff
@@ -1,9 +1,36 @@
 package telegram
 
 type Chat struct {
-	ID        int    `json:"id"`
-	Title     string `json:"title"`
-	Type      string `json:"type"`
```

Comment:
Just add a enum type `ChatType` and some const for exists value, like the code in other packages.